Makefile

```diff
@@ -41,7 +41,15 @@ osx-init:
 	export GOPATH=~/go/
 	export PATH=$PATH:~/go/bin
 	make init
-
+osx-init-travis:
```

Comment:
No need to have this in the Makefile - can just inline in the travis-ci.sh script?

Reply:
OK. Revising and pushing.